src/status_im/contexts/chat/messenger/messages/contact_requests/bottom_drawer/view.cljs

```diff
@@ -70,4 +72,8 @@
                        (i18n/label :t/contact-request-chat-received {:name primary-name})
 
                        contact-request-pending?
-                       (i18n/label :t/contact-request-chat-pending))}]]))
+                       (i18n/label :t/contact-request-chat-pending)
+                       
+                       contact-request-dismissed?
+                       (i18n/label :t/contact-request-chat-add {:name primary-name}))}]]))
+                       
```

Comment:
Adding a label because we were not handling the dismissed state and that was causing some schema errors